src/Cake.NuGet/Install/NuGetPackageInstaller.cs

```diff
@@ -96,6 +100,18 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
                 return Array.Empty<IFile>();
             }
 
+            var packagePath = GetPackagePath(root, package.Package);
+            if (packagePath != null)
+            {
+                // Fetch available content from disc.
+                var content = _contentResolver.GetFiles(packagePath, package, type);
```

Comment:
As no packages have been installed, we also do not know of any dependencies at this point. Thus, loading dependencies will not work.